build.gradle

```diff
@@ -97,6 +97,8 @@ ext {
     awsCommand = "aws"
 
     sdkZip = file("${buildDir}/java-sdk-${version}.zip")
+
+    swaggerVersion = "1.5.16"
```

Comment:
Would it make sense to use this dependency management plugin instead? https://github.com/spring-gradle-plugins/dependency-management-plugin